documentation/Extension_Point_WebScripts.md

```diff
@@ -198,6 +198,18 @@ protected void handleIllegalArgument(IllegalArgumentException exception, WebScri
     response.setStatus(400);
 }
 ```
+This handler method must be defined in the same WebScript that will throw the Exceptions to be handled.
```

Comment:
I would rephrase it a bit to use the correct terminology: 
`... in the same @WebScript annotated class ...`
?

However if it's more clear like it is now, it's good for me. Other opinions welcome.

Reply:
I'd say your suggestion is more clear, agreed.